### DISCUSSION TITLE
Unable to launch dashboard

### DISCUSSION THREAD

**Discussion by satya-h** [chaos-mesh/chaos-mesh#1333] [Q&A] [answered]
Hi, 
I am trying to use chaos-mesh for our project. I followed the steps given in documents. It has been installed successfully. All daemon, dashboard, controller pods are in running state. But when I hit the dashboard url it is redirecting to /dashboard url. 
It is showing as /dashboard url is not available in server. 

#I am not allowed to share any screenshots as per our company policy. 

Kindly help me

**Comment by zhouqiang-cl** [comment]
@g1eny0ung PTAL

**Comment by g1eny0ung** [comment, marked as answer]
First, what version of Chaos Mesh u use? The redirecting is normal behavior.

From the information you gave, I can't tell why the dashboard UI is invalid.

**Reply by satya-h** [reply]
@g1eny0ung 
I am using app version v1.0.3 chart version v0.3.3.

![IMG_20201229_110308](https://user-images.githubusercontent.com/76690806/103260942-98732500-49c5-11eb-996f-3e444c9504bc.jpg)

![IMG_20201229_105959](https://user-images.githubusercontent.com/76690806/103260826-1aaf1980-49c5-11eb-8b5a-af03a3ec5b0e.jpg)

**Reply by g1eny0ung** [reply]
Know about it.

I think the dashboard has a `NodePort` service default in `v1.0.3`, If you haven’t done any additional operations. For example in my local machine:

```
kubectl get svc -n chaos-testing
NAME                            TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                           AGE
chaos-daemon                    ClusterIP   None             <none>        31767/TCP,31766/TCP               3h4m
chaos-dashboard                 NodePort    10.106.206.179   <none>        2333:30346/TCP                    3h4m
chaos-mesh-controller-manager   ClusterIP   10.103.153.218   <none>        10081/TCP,10080/TCP,443/TCP       3h4m
chaos-mesh-dns-server           ClusterIP   10.104.20.186    <none>        53/UDP,53/TCP,9153/TCP,9288/TCP   3h4m
```

Run by `kubectl get nodes --output wide` to check the node EXTERNAL-IP, then access the dashboard with PORT 30346.

Or you can use `kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333` to access it temporarily.

**Reply by satya-h** [reply]
I tried all possible ways. Still it is same. 
Can you tell me the latest version. I will try that one.

**Reply by g1eny0ung** [reply]
In Chaos Mesh repo, run `helm install chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set dashboard.create=true` to install the latest version.

But I think the problem you encounter should have nothing to do with the version 🤔.

**Reply by STRRL** [reply]
Hi @satya-h , I noticed that you are trying to access chaos-dashboard by a domain name like `ieatenmc33.athtem.eei.ericsson.se` and port `80`. It seems like using a load-balancer or ingress.

And from the output of `kubectl get svc -n chaos-testing`, the `chaos-dashboard` is a `NodePort` service. You could try http://10.106.206.179:2333

**Reply by satya-h** [reply]
Thank you very much guys. I tried with nodeport ip and svc port. It is accessible.
